data/migrations/20181130102421_create_project_table.js

```diff
@@ -0,0 +1,13 @@
+
+exports.up = function(knex, Promise) {
+  return knex.shema.createTable('users', function(table){
+    table.increments();
+      table.string('name').notNullable();
+      table.string('description').notNullable();
+     table.boolean('completed').notNullable().defaultTo(false);
+  })
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema.dropTable('users');
```

Comment:
this looks good so far! Let's just get the helpers and the index.js set up and knock out these API calls